conf/ocs_version/ocs-4.2-alpha.yaml

```diff
@@ -0,0 +1,14 @@
+---
+# This config file is for 4.2 bulds which have only alpha in the channel
```

Comment:
s/bulds/builds/

Reply:
Finally as we agreed on different solution this config was deleted.